themes/bootstrap3/js/common.js

```diff
@@ -271,18 +271,20 @@ var VuFind = (function VuFind() {
   };
 
   function setupQRCodeLinks(_container) {
-    var container = _container || $('body');
-
-    container.find('a.qrcodeLink').on('click', function qrcodeToggle() {
-      var holder = $(this).next('.qrcode');
-      if (holder.find('img').length === 0) {
-        // We need to insert the QRCode image
-        var template = holder.find('.qrCodeImgTag').html();
-        holder.html(template);
-      }
+    var container = _container || document.body;
+    var qrcodeLinks = container.querySelectorAll('a.qrcodeLink');
+    qrcodeLinks.forEach((link) => {
+      link.addEventListener('click', () => {
+        var holder = this.nextElementSibling;
+        if (holder.find('img').length === 0) {
+          // We need to insert the QRCode image
+          var template = holder.find('.qrCodeImgTag').html();
+          holder.html(template);
+        }
+      });
     });
   }
-
+  
```

Comment:
Looks like some unnecessary whitespace was added here. Might want to check your IDE's "trim whitespace" setting.